draft-wing-settle-public-key-hash.md

```diff
@@ -233,7 +233,20 @@ connection is made to that address, those are also considered
 
 # Security Considerations
 
-TODO: write more on security considerations
+Due to operational challenges in key rotation, some servers may need to
+maintain static public/private key pairs over long periods. This introduces a
+tradeoff: while static keys expose servers to risks of private key compromise
+and active attacks (e.g., traffic interception or server impersonation), they
+still provide better security than unencrypted communication. The current approach
+to use unencrypted communication to local servers—is highly vulnerable to passive
+attacks. Another option, conditioning users to validate self-signed certificates,
+is both error-prone and impractical.
```

Comment:
I'd rather not get into the problems of self-signed certificates; it requires way more text and is perhaps part of a requirements document or a 'motivation' document or section.

```suggestion
attacks. 
```